src/back/glsl.rs

```diff
@@ -1,10 +1,12 @@
 use super::{BorrowType, MaybeOwned};
 use crate::{
+    proc::{Interface, Visitor},
     Arena, ArraySize, BinaryOperator, BuiltIn, Constant, ConstantInner, DerivativeAxis, Expression,
     FastHashMap, Function, FunctionOrigin, GlobalVariable, Handle, ImageClass, Interpolation,
     IntrinsicFunction, LocalVariable, MemberOrigin, Module, ScalarKind, ShaderStage, Statement,
     StorageAccess, StorageClass, StorageFormat, StructMember, Type, TypeInner, UnaryOperator,
 };
+use log::warn;
```

Comment:
note that in other places we just use the full path for logging, i.e. `log::warn!("...")`